src/content/en/2022/seo.md

```diff
@@ -1,22 +1,754 @@
+* 10% of desktop and mobile pages had `<meta name="description>` tags containing 35 words.
+* 10% of desktop and mobile pages had `<meta name="description>` tags containing 232 characters.
+
+For the most part, these stats are relatively unchanged from last year.
+
+#### Header tags
+
+Heading elements  (`<h1>`, `<h2>`...) are important parts of a page’s structure since they help organize the content on the page. Heading elements are not a direct ranking factor, but they  can help Google better understand the content found on the page.
+
+{{ figure_markup(
+  image="has-h-elements.png",
+  caption="66% of sites have implemented an H1.",
+  description="A column charting showing usage of H1 tags through H4 tags.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSZxId5PPpTHp4835BI-rt4HqZVrhn1UiF0xcOrVUytVs6j262ZqFrY5e1FXZibAGDY6gWnZ0uG3WZC/pubchart?oid=46522676&format=interactive",
+  sheets_gid="1564568239",
+  sql_file="seo-stats.sql"
+  )
+}}
+
+The trends around implementation of headings by type in 2022 closely match those from 2021, with just a few small differences. For example, 71.9% of mobile pages utilized an h2 in 2021 while 73.02% did in 2022.
+
+Another trend that has carried over is the discrepancy in usage between the h1 and h2. While 72.7% of desktop pages implement an h2, only 65.8% use an h1 (with similar numbers reflected on mobile).
+
+Although there is no  definitive explanation for this, one possible reason is that the h1 is often placed above any content. It’s not essential for the natural flow of the content. However, without the h2, there could be a long flow of unstructured content.
+
+{{ figure_markup(
+  image="nonempty-h-elements.png",
+  caption="59% of sites have an H1 tag that is not empty.",
+  description="A column charting showing non-empty usage of H1 tags through H4 tags.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSZxId5PPpTHp4835BI-rt4HqZVrhn1UiF0xcOrVUytVs6j262ZqFrY5e1FXZibAGDY6gWnZ0uG3WZC/pubchart?oid=1398855380&format=interactive",
+  sheets_gid="1564568239",
+  sql_file="seo-stats.sql"
+  )
+}}
+
+Overall, much like 2021’s stats, there are relatively few empty H elements found on pages. Additionally, there is little discrepancy between the desktop and mobile data.
+
+There is divergence, however, with the h1.  While 65.8% of pages contained an h1 element, 58.5% contained a non-empty h1 element. That’s a 7.3 percentage point difference. Contrast that with the h2, which has just a 1.5 percentage point difference.  As noted in the 2021 Web Almanac, this may be a result of the many websites that wrap logo-images in the h1 element on homepages.
+
+### Image attributes
+
+The primary purpose of the alt attribute on the `<img>` element is accessibility. Alt attributes also assist search engines rank specific assets in image search.
+
+[Insert the 3 charts related to present, blank, and missing alt attributes]
```

Comment:
Can this be removed as looks like charts are there?

Reply:
Yes line 458 can be removed

Reply:
Removed